en/about.markdown

```diff
@@ -1,18 +1,66 @@
 ---
 layout: page
-title: About
+title: What is code curious?
```

Comment:
Keep in mind, the title is what is used for the "header link" text.

Reply:
I realized but I actually want it to be `about` on the header but on the page itself there should not be a big `about`.. how can I not have it show on the page itself?